rfc/ipni-w3c.md

```diff
@@ -93,21 +94,19 @@ curl https://cid.contact/cid/bafybeicawc3qwtlecld6lmtvsndimoz3446xyaprgsxvhd3aap
 ]}]}
 ```
 
-web3.storage publishes the blocks it can provide by encoding a batch of multihashes as an IPLD object and writing it to a bucket as an `Advertisement`, addressed by it's multihash.
+web3.storage will publish the blocks it can provide by encoding a batch of multihashes and information needed to retrieve tha data into an IPNI `Advertisement`. The advertisement is referred to by its CID.
 
-An `Advertisement` includes `Provider` info which claims that the batch of multihashes are available via bitswap or HTTP, and are signed by the providers PeerID private key; Each advert is a claim that this peer will provide that batch of multihashes.
+An `Advertisement` includes `Provider` info which claims that the batch of multihashes are available via some protocol (e.g. bitswap or HTTP), and are signed by the provider's PeerID private key; Each advertisement advertises to IPNI that the stored data, specified by multihashes, is available from the provider.
```

Comment:
Need to think about this...

The w3 service is the one publishing an advertisement. I thought we were going to have all of the location claims created by the w3s service, meaning that is where the service understands the data to be located.

We can certainly include location claims created by the user. I was under the impression that we did not want to do this since the service has authority over the location, or at least over verifying the data was at the location.  @hannahhoward any opinion here?

FYI, the above mention of the signature has nothing to do with the location claim or claim bundle, it is just a mechinism to ensure the advertisement was legitimately added to the ad chain.